implementations.md

```diff
@@ -0,0 +1,11 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCI runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCI
```

Comment:
For consistency, probably capitalize the first word after the dash in each case (so “Reference implementation of OCI runtime” for runC).  And if we're setting runV apart based on the underlying technology, it would be nice to list runC's supported systems too, so folks know when to chose one over the other.